src/form/input/Input.tsx

```diff
@@ -8,6 +8,7 @@ export type InputProps = Omit<
   "disabled" | "name"
 > & {
   name: string;
+  type?: "text" | "color";
```

Comment:
```suggestion
  type?:
    | "checkbox"
    | "button"
    | "color"
    | "date"
    | "datetime-local"
    | "email"
    | "file"
    | "hidden"
    | "image"
    | "month"
    | "number"
    | "password"
    | "radio"
    | "range"
    | "reset"
    | "search"
    | "submit"
    | "tel"
    | "text"
    | "time"
    | "url"
    | "week";
```

Let's provide all available options